torch2trt/converters/view.py

```diff
@@ -1,21 +1,23 @@
 from torch2trt.torch2trt import *
+# from torch2trt.shape_conversion import *
 from torch2trt.module_test import add_module_test
 
 
-@tensorrt_converter('torch.flatten')
-@tensorrt_converter('torch.Tensor.reshape')
 @tensorrt_converter('torch.Tensor.view')
-@tensorrt_converter('torch.Tensor.squeeze')
-@tensorrt_converter('torch.Tensor.unsqueeze')
-@tensorrt_converter('torch.Tensor.flatten')
-@tensorrt_converter('torch.squeeze')
-@tensorrt_converter('torch.unsqueeze')
+@tensorrt_converter('torch.Tensor.reshape')
 def convert_view(ctx):
     input = ctx.method_args[0]
-    input_trt = add_missing_trt_tensors(ctx.network, [input])[0]
```

Comment:
Is this okay to remove? I'm currently running into a situation where I need to attach a TRT tensor to the input because it doesn't already exist...

Reply:
Just did some testing, looks like we need to call this and similar methods because member variable tensors of the class do not automatically get converted to TRT (ie. gets the `._trt` attribute) until they are encountered in a converter/operation. Whereas most tensors get this attribute by default on the output of a converter (ie. `output._trt`), this is not the case for member variables because their first encounter is in the op in which they are used, at which point they might yet be initialized with a `._trt` attribute.

Reply:
Hi @chaoz-dev ,

I ran into this issue as well while optimizing timm's maxvit model.

I addressed this in the following branch by deferring adding constants until a non-unary operation is found that has at least one non-constant trt input.

This is done in this branch, and may address this issue for most converters.

https://github.com/NVIDIA-AI-IOT/torch2trt/compare/master...jaybdub:torch2trt:maxvit

Reply:
I'll need to test, but I can re-base this PR on these changes.